dom.bs

```diff
@@ -639,10 +639,18 @@ dictionary EventInit {
 <p>An {{Event}} object is simply named an <dfn export id=concept-event>event</dfn>. It allows for
 signaling that something has occurred, e.g., that an image has completed downloading.</p>
 
+<p>An <a>event</a> has an associated <dfn export for=Event>relatedTarget</dfn> (an
+{{EventTarget}} object). Unless stated otherwise it is null.</p>
+
+<p>An <a>event</a> has an associated <dfn export>set the relatedTarget</dfn> algorithm,
+which takes an {{EventTarget}} <var>relatedTarget</var>. Unless specified otherwise it does
+nothing.</p>
```

Comment:
Good question. This is inspired by "get the parent", however, I expect that there is only one implementation. UI Events and Touch Event would actually do the same thing. They just store a given argument to their relatedTarget attribute.

I just would like to avoid mentioning relatedTarget *attribute* in DOM Standard because Event interface does not have relatedTarget attribute. If it is okay to mention relatedTarget attribute in DOM Standard, I would remove "set the relatedTarget" abstraction.

For example, can we say the following?
> Set event’s relatedTarget attribute to *relatedTarget*, if event has a relatedTarget attribute. Otherwise, do nothing.